pkg/operator/storageclasscontroller/storageclasscontroller.go

```diff
@@ -105,15 +128,26 @@ func (c *StorageClassController) syncStoragePolicy(ctx context.Context, connecti
 		return "", checks.MakeClusterCheckResultPass()
 	}
 
-	infra := apiDeps.GetInfrastructure()
+	// if we are running the checks after creating the policy successfully
+	// then lets run checks less frequently.
+	if !time.Now().After(c.nextCheck) && len(c.policyName) > 0 {
+		klog.Infof("Returning without running any checks")
```

Comment:
This log statement looks like it will end up flooding the logs with this message. Could it be a debug log instead to reduce noise?